src/vivarium_nih_us_cvd/constants/data_values.py

```diff
@@ -1,37 +1,65 @@
 from datetime import datetime
+from typing import NamedTuple
 
 ############################
 # Disease Model Parameters #
 ############################
 
-REMISSION_RATE = 0.1
-MEAN_SOJOURN_TIME = 10
+# REMISSION_RATE = 0.1
+# MEAN_SOJOURN_TIME = 10
+
+
+################################
+# Healthcare System Parameters #
+################################
+
+
+class __VisitTypes(NamedTuple):
+    """visit types to be observed"""
+    EMERGENCY: str = "emergency"
+    SCHEDULED: str = "scheduled"
+    MISSED: str = "missed"
+    BACKGROUND: str = "background"
+
+    @property
+    def name(self):
+        return "visit_types"
+
+
+VISIT_TYPES = __VisitTypes()
+
+VISIT_TYPE_COLUMN = "visit_type"
+SCHEDULED_VISIT_DATE_COLUMN = "scheduled_date"
+MISS_SCHEDULED_VISIT_PROBABILITY_COLUMN = "miss_scheduled_visit_probability"
+
+MISS_SCHEDULED_VISIT_PROBABILITY_MIN = 0.05
+MISS_SCHEDULED_VISIT_PROBABILITY_MAX = 0.35
 
 
 ##############################
 # Screening Model Parameters #
 ##############################
 
-PROBABILITY_ATTENDING_SCREENING_KEY = "probability_attending_screening"
-PROBABILITY_ATTENDING_SCREENING_START_MEAN = 0.25
-PROBABILITY_ATTENDING_SCREENING_START_STDDEV = 0.0025
-PROBABILITY_ATTENDING_SCREENING_END_MEAN = 0.5
-PROBABILITY_ATTENDING_SCREENING_END_STDDEV = 0.005
+# PROBABILITY_ATTENDING_SCREENING_KEY = "probability_attending_screening"
```

Comment:
is there a reason these are commented out vs. deleted?

Reply:
I just know (well - think) that I'll need to pull them in on the next run. They are actually there from the template.